client/flutter/lib/l10n/intl_kr.arb

```diff
@@ -0,0 +1,26 @@
+{
+    "worldHealthOrganization": "세계보건기구",
+    "learnMore": "더 읽어보기",
+    "protectYourself": "자신을 지키세요",
+    "checkYourHealth": "건강을 체크하세요",
+    "localMaps": "지역 지도",
+    "shareTheApp": "앱 공유하기",
+    "healthCheck": "건강 체크",
+    "feelingDistressed": "아픔을 느끼시나요?",
```

Comment:
This translation strongly implies physical pain rather than mental, would you please consider using the term `근심/걱정/불안/스트레스`? i.e. `극심한 스트레스 증상이 있습니까?`

Reply:
Thanks, I didn't realize that it doesn't mean physical pain.